hystrix-microservice-provider/src/main/java/io/silverware/microservices/providers/hystrix/configuration/AnnotationScanner.java

```diff
@@ -35,76 +35,93 @@
 import java.util.Arrays;
 import java.util.HashMap;
 import java.util.HashSet;
+import java.util.List;
 import java.util.Map;
 import java.util.Optional;
 import java.util.Set;
+import javax.enterprise.inject.spi.InjectionPoint;
 
 /**
  * Scans annotations and creates Hystrix service configuration.
  */
 public class AnnotationScanner {
 
+   private static final List<Class<? extends Annotation>> HIGH_LEVEL_ANNOTATIONS = Arrays.asList(Cached.class, CircuitBreaker.class, Fail.class, ThreadPool.class, Timeout.class);
```

Comment:
You are doing contains queries on this collection and I think that `Set` is more suitable for this.